### ISSUE TITLE
Replacing the copyright block binding

### ISSUE THREAD

**Issue by carolinan** [WordPress/twentytwentyfive#571]
**Is your feature request related to a problem? Please describe.**
Cross posting this ticket here so that it is not missed
https://core.trac.wordpress.org/ticket/62143

I believe the actionable points for now are:

- Replace the copyright footer text with the theme name

**Comment by carolinan** [comment]
What about keeping the copyright date binding in the theme, but letting the user decide if they want to use it?
And change the date to the date of the first published post or page, not the current year.

**Comment by 2ndkauboy** [comment]
I read the conversation about this issue. I believe the issue with "naming the copyright holder" would still persist, right? Then not using a "copyright footer text" but only the theme name might be best.

**Comment by carolinan** [comment]
@beafialho Are there any places in the design that would benefit from only displaying one category or tag?

With the post terms block, there is no way to limit how many categories or tags that are showing.
With the block bindings, we could limit it to one. The problem is that the user would not be able to select _which_ single category or tag that shows.

In the editor the placeholder text could be something like "first category".

**Comment by beafialho** [comment]
> [@beafialho](https://github.com/beafialho) Are there any places in the design that would benefit from only displaying one category or tag?
> 
> With the post terms block, there is no way to limit how many categories or tags that are showing. With the block bindings, we could limit it to one. The problem is that the user would not be able to select _which_ single category or tag that shows.
> 
> In the editor the placeholder text could be something like "first category".

Interesting question @carolinan. Perhaps the News blog templates would be good places to try (only the Blog templates, not the Single post templates though). Given that these designs contemplate one category for each post as a way to reduce the amount of information on the homepage.

Maybe:
- News blog home
- News blog with featured posts grid
- News blog with sidebar

**Comment by carolinan** [comment]
I have not tried it inside a query loop, will check if my test code works.

**Comment by carolinan** [comment]
@beafialho @juanfra I tried using the block binding to display the category on these templates, but the problem is that it does not inherit the style from the terms block, since it uses a paragraph.
So when I switch style variations, the letter spacing and font weight do not match the current styles.

I also found that in the editor, where the placeholder shows, the styling is also different because there is no link to the category archive. The link is only used on the front.
